subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CopyTaskIntegrationSpec.groovy

```diff
@@ -21,11 +21,14 @@ import org.gradle.api.plugins.ExtensionAware
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 import org.gradle.integtests.fixtures.TestResources
 import org.gradle.integtests.fixtures.ToBeFixedForConfigurationCache
+import org.gradle.test.fixtures.file.TestFile
 import org.gradle.util.Matchers
 import org.gradle.util.internal.ToBeImplemented
 import org.junit.Rule
 import spock.lang.Issue
 
+import static org.apache.commons.io.FilenameUtils.separatorsToUnix
```

Comment:
❌ Our fixture for this is `TextUtil.normaliseFileSeparators`

Reply:
Good to know - I looked and somehow didn't find this.